src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  gap: 4px;
+
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--4 &__star:last-child {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
So i need to create a loop for stars count, so it would be possible to change the number of stars review, isn`t it? But what about stars task styles, i cant figure it out.

Reply:
Please clarify more information about stars task styles. You should reuse it from stars task and put styles to `stars.scss` file